x11-misc/polybar/polybar-3.2.1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Touching a stable ebuild is not wanted.

Reply:
Do you mean a revision bump would be in order, e.g. `x11-misc/polybar/polybar-3.2.1-r1.ebuild`?

Reply:
yes